### PR TITLE
Final updates

### PR DESCRIPTION
**Pull request by ccondra**
The purpose of this pull request is to submit final dotmailer edits for the 2.3.3 release.
Sandbox:
- Open Source: https://docs.magedevteam.com/253/m2/ce/user_guide/
- Commerce: https://docs.magedevteam.com/253/m2/ee/user_guide/
- B2B: https://docs.magedevteam.com/253/m2/b2b/user_guide/

1) Reworked format of Change Log entries for Customer Engagement field desc 
2) removed word "now" from What's New In This Release

## Affected Magento editions

- [ x  ] Open Source
- [ x]  Commerce
- [ x ] B2B

### CODE REVIEW

**Comment by ccondra** [comment]
I don't know why the Jenkins status is pending, because I generated the sandbox output.

File: src/marketing/engagement-cloud.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Engagement Cloud, powered by [dotdigital][1]{: target="_blank"}, gives you the a
 * [Generate reports]({{ site.baseurl }}{%- link reports/customer-engagement.md -%})
 
 ## Changes In the Latest Release
```
Comment (dshevtsov) [inline src/marketing/engagement-cloud.md]: Add a blank line to separate the heading and text of the section.

Comment (ccondra) [reply src/marketing/engagement-cloud.md]: Fixed.

**Comment by ccondra** [comment]
running tests

**Pull request merged**